esp/src/eclwatch/stub.js

```diff
@@ -154,9 +155,13 @@ define([
 
                 document.title = widget.getTitle ? widget.getTitle() : params.Widget;
 
-                if (cookie("ESPSessionTimeoutSeconds")) {
+                if (sessionIsActive) {
+                    if (!cookie("ECLWatchUser")) {
+                        dojo.cookie("ECLWatchUser", "true");
```

Comment:
What is this cookie used for?  My concern is that it can be set and there might be no valid session.

Reply:
We check to verify if there is a valid session before setting the cookie. The issue is that ESP will try to redirect to Login.html but, due to the cache of the browser it sends us into a "session-less" ECLWatch repeatedly.